<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CorrectUIClass</class>
 <widget class="QMainWindow" name="CorrectUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1395</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CorrectUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>331</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="clearButtom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadImageButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>导入图像</string>
     </property>
    </widget>
    <widget class="QListWidget" name="imageNameList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>191</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="loadParameterButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>导入角度参数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pathEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="correctButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>矫正</string>
    </property>
   </widget>
   <widget class="QLabel" name="showImageLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>580</width>
      <height>540</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="showImageLabel1">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>10</y>
      <width>580</width>
      <height>540</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>431</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>图像系数a</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="parmEdit">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>430</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="correctui.qrc"/>
 </resources>
 <connections/>
</ui>
